components/director/internal/domain/apptemplate/graphqlizer.go

```diff
@@ -268,8 +263,31 @@ func (g *Graphqlizer) VersionInputToGQL(in graphql.VersionInput) (string, error)
 	}`)
 }
 
+func (g *Graphqlizer) marshal(obj interface{}) string {
```

Comment:
Can we use imported graphqlizer from tests? Or can we move graphqlizer to director and then import it in tests?
Please also let known corresponding teams which use copied Graphqlizer.